Advanced Tutorial.md

```diff
@@ -0,0 +1,211 @@
+# Advanced API Blueprint Tutorial
+
+Welcome to the advanced API Blueprint tutorial! This tutorial will take you through advanced topics like JSON Schema, request and response attributes, data structures and relation types.
+
+This tutorial assumes that you have read the [API Blueprint Tutorial](./Tutorial.md).
+
+## JSON Schema
+
+Action request and response bodies can have associated schemas that describe the allowed structure of the body content. JSON bodies are typically described with [JSON Schema](http://json-schema.org/). Given a simple JSON response body we can describe the structure of the response with JSON Schema in a `+ Schema` section:
+
+```apib
+## Random Numbers [/random]
```

Comment:
I think it would be nice to have a tutorial that keeps the same domain along all of the examples so it's easier to follow. Instead of jumping from a random number generator to the polls app, we could make both examples related to the polls app.